control-plane/roles/monitoring/templates/alerts_ingress.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: alerts-basic-auth
+  namespace: monitoring
+type: Opaque
+data:
+  auth: 
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: monitoring-ingress-alerts
+  annotations:
+    nginx.ingress.kubernetes.io/auth-type: basic
+    nginx.ingress.kubernetes.io/auth-secret: alerts-basic-auth
+    nginx.ingress.kubernetes.io/auth-realm: 'Authentication Required - alertmanager'
+    cert-manager.io/cluster-issuer: letsencrypt-prod
```

Comment:
Annotations for cert-management should be passable through a variable like `monitoring_additional_ingress_annotations`.

Reply:
https://github.com/metal-stack/metal-roles/pull/115/commits/9d8c2c208daa596dfffe357fa75874ab9972daf3

Reply:
https://github.com/metal-stack/metal-roles/commit/e1fa4fcfb4ec89e72589ee3ee61d2f2d211b1571